content/_en/partners/faq.md

```diff
@@ -0,0 +1,188 @@
+        content: >-
+            #### For your agency:
+
+            - Implementation support for your team from integration to launch
+
+            - Troubleshooting after launch and user support
+
+            - Troubleshooting after launch and user support
+
+            - Enhanced fraud detection and monitoring
+
+            - High availability and uptime
+
+            - Secure two-factor authentication (2FA) backed by a FedRAMP Moderate ATO, which helps your agency meet the President’s “Executive Order on Improving the Nation’s Cybersecurity”
+
+            - Reduced costs through economies of scale across government
+
+            - A platform that stays up to date with current authentication and identity policies, technologies, and standards, without additional effort required from your agency
+
+            #### For your end-users:
+
+            - One account to access all their websites and applications - eliminating the need to remember multiple passwords and usernames
+
+            - A secure and private authentication experience
+
+            - An intuitive and well designed user experience
+
+            - Simple account management Online help center and robust customer support team
+    -
+        id: a-2
+        title: Who has Login.gov partnered with?
+        content: >-
+            Login.gov has over 40 agency partners. Our product is integrated with over 200 live applications and services including 12 Cabinet level agencies such as the Department of Defense, Department of Homeland Security, Department of Energy, and the Executive Office of the President. 
+    -
+        id: a-3
+        title: How many people have signed up to use Login.gov?
+        content: >-
+            Over 40 million people have signed up to use Login.gov, across all applications, with over 135 million sign-ins annually.
+    -
+        id: a-4
+        title: Is Login.gov a federal agency?
+        content: >-
+            Login.gov is not a stand alone federal agency. We are a program of the General Services Administration, an agency of the U.S. federal government. The program is run by the Technology Transformation Services, a group that leads the digital transformation of the federal government by helping agencies build, buy, and share technology that allows them to provide more accessible, efficient, and effective products and services for the American people.
+    -
+        id: a-5
+        title: Does Login.gov partner with state and local governments?
+        content: >-
+            Yes, Login.gov partners with state and local governments. State and local governments need simple and secure solutions to help the public access federally funded services and resources. With this partnership, those agencies can leverage Login.gov to create a seamless and secure sign in experience for the public to access these resources. <a href="{{ '/partners/state-and-local/' | locale_url }}" class="caret">Learn more about the path to partnership</a>
```

Comment:
The "Learn more about the path to partnership" link is broken. Jekyll doesn't like evaluating Liquid ( {{ }} ) in Markdown files. Try this: https://github.com/18F/identity-site/pull/774/files#diff-a8215f12a5c1873523fd381831ac49953c49f4a75e5f92a5b1818f678252079aR27 and make sure you add the display: inline to the p proceeding your link: https://github.com/18F/identity-site/pull/774/files#diff-af4f96e461f626555c17161f4baca330943a780e92726fec6833137d71a17fe4R33

Reply:
This is sort of crushing.  I'll have to refactor the entire page to get this link into html.

Reply:
oh no, really?

Reply:
oh bc you're iterating for the accordion 😞